src/api/route-services/projects.js

```diff
@@ -18,6 +18,7 @@ const experimentService = new ExperimentService();
 class ProjectsService {
   constructor() {
     this.tableName = `projects-${config.clusterEnv}`;
```

Comment:
to make it more obvious, would it be better to rename this to `projectsTableName`?